system/assets/js/admin/specific/datamanager/object/batchUpdateButton.js

```diff
@@ -21,4 +21,25 @@
 		} );
 	}
 
+	// select the target node
+	var target = document.querySelector('table');
```

Comment:
Is there any reason this is not `$( ".object-listing-table" ).get( 0 )`? Seems odd to watch _all_ tables?